end_to_end_test/lib/values_nnbd.dart

```diff
@@ -7,6 +7,7 @@ import 'package:built_collection/built_collection.dart';
 import 'package:built_value/built_value.dart';
 import 'package:built_value/serializer.dart';
 import 'package:end_to_end_test/enums_nnbd.dart' as using_import_as;
+import 'package:end_to_end_test/values.dart';
```

Comment:
It should work to remove this import, I think?

The presubmit is complaining because it's a "null safe" -> "legacy" import which isn't allowed.

Reply:
Oh my IDE must have added this while I was poking around without noticing :sweat_smile:

And I was wondering why the tool gave me such a 'complex' exit code (at least it was green).
Sorry for not noticing :)